docs/sdk/src/reference_docs/frame_currency.rs

```diff
@@ -1,8 +1,51 @@
-//! FRAME Currency Abstractions and Traits
+//! # FRAME Currency Abstractions and Traits
 //!
-//! Notes:
+//! Currency related traits in FRAME provide standardized interfaces for implementing various
+//! currency functionalities. These traits enable developers to create, transfer, and manage
+//! different forms of digital assets within the blockchain environment, ensuring that the economic
+//! aspects of the chain are robust and adaptable to different use cases.
 //!
-//! - History, `Currency` trait.
-//! - `Hold` and `Freeze` with diagram.
-//! - `HoldReason` and `FreezeReason`
-//! 	- This footgun: <https://github.com/paritytech/polkadot-sdk/pull/1900#discussion_r1363783609>
+//! ## The Currency Trait
+//!
+//! The [`Currency`](../../../frame_support/traits/tokens/currency/index.html) trait was initially
+//! introduced in Substrate to manage the native token balances. This trait was later deprecated in
```

Comment:
Reason: the interface is only sensible in a system that has only one currency type. There is no such thing in it as "id".